include/SDL3/SDL_gpu.h

```diff
@@ -492,6 +773,12 @@ typedef enum SDL_GPUSwapchainComposition
     SDL_GPU_SWAPCHAINCOMPOSITION_HDR10_ST2048
 } SDL_GPUSwapchainComposition;
 
+/**
+ * Specifies a backend API supported by SDL_GPU.
+ * Only one of these will be in use at a time.
+ *
+ * \since This enum is available since SDL 3.0.0
+ */
 typedef enum SDL_GPUDriver
```

Comment:
This enum is being removed, right?

Reply:
I think you're right, but we can cross that bridge when we get there.